src/scss/components/_button.scss

```diff
@@ -227,3 +227,11 @@ $button-padding-h: $grid-margin-2;
     }
   }
 }
+
+.openforms-theme {
+  .utrecht-button {
+    @include bem.modifier('disabled') {
+      cursor: not-allowed;
```

Comment:
This is possible as design token:

```json
{
  "utrecht": {
    "action": {
      "disabled": { "cursor": { "value": "not-allowed" } }
    }
  }
}
```

Reply:
ah I went looking for existing design tokens but couldn't find them quickly enough, thanks for the pointers! I'll update our theme in the design tokens repo and get rid of these overrides.